docs/rules/no-restricted-properties.md

```diff
@@ -39,6 +39,32 @@ Multiple object/property values can be disallowed, and you can specify an option
 }
 ```
 
+If the object name is omitted, the property is disallowed for all objects:
+
+```json
+{
+    "rules": {
+        "no-restricted-properties": [2, {
+            "property": "__defineGetter__",
+            "message": "Please use Object.defineProperty instead."
+        }]
+    }
+}
+```
+
+If the property name is omitted, accessing any property of the given object is disallowed:
+
+```json
+{
+    "rules": {
+        "no-restricted-properties": [2, {
+            "object": "require",
+            "message": "Please call require() directly."
```

Comment:
Should we omit the "message" property in these examples for clarity, since what you're exemplifying is the object/property option?